wagtail/admin/wagtail_hooks.py

```diff
@@ -998,6 +998,23 @@ def register_editors_guide_menu_item():
     )
 
 
+@hooks.register("register_help_menu_item")
+def register_keyboard_shortcuts_menu_item():
+    return MenuItem(
+        _("Shortcuts"),
+        icon_name="regex",
+        order=1200,
+        attrs={
+            "role": "button",
+            "data-a11y-dialog-show": "keyboard-shortcuts-dialog",
+            "data-action": "w-action#noop:prevent:stop",
+            "data-controller": "w-action",
+        },
+        name="keyboard-shortcuts-trigger",
+        url="#",
+    )
```

Comment:
It would be very cool if opening the dialogue could be a shortcut too. `command + /` is common.

Reply:
Yep. It's on the plans in https://github.com/wagtail/wagtail/issues/3949 - it's dependant on a different bit of work making it easier to add keyboard shortcuts with just data attributes. 

See https://github.com/wagtail/wagtail/issues/10167 & https://github.com/wagtail/wagtail/pull/11740